core/templates/dev/head/services/AssetsBackendApiService.ts

```diff
@@ -91,10 +91,26 @@ oppia.factory('AssetsBackendApiService', [
                          window.MozBlobBuilder ||
                          window.MSBlobBuilder;
           if (exception.name === 'TypeError' && window.BlobBuilder) {
-            var blobBuilder = new BlobBuilder();
-            blobBuilder.append(data);
-            assetBlob = blobBuilder.getBlob(assetType.concat('/*'));
+            try {
```

Comment:
I'm not sure but is this error happening in some browser because we aren't adding the 2nd argument on line 86?

Reply:
oh, that line is already inside try catch.

Reply:
Second argument is optional, so I don't think that is the reason. Also, the issue doesn't mentions any browser, so we cannot be sure about that too.